test/mocks/ECDSAServiceManagerMock.sol

```diff
@@ -19,4 +27,18 @@ contract ECDSAServiceManagerMock is ECDSAServiceManagerBase {
     ) public virtual initializer {
         __ServiceManagerBase_init(initialOwner, rewardsInitiator);
     }
+
+    function createOperatorSets(uint32[] memory) external {}
+
+    function registerOperatorToOperatorSets(
+        address operator,
+        uint32[] calldata operatorSetIds,
+        ISignatureUtils.SignatureWithSaltAndExpiry memory operatorSignature
+    ) external {}
+
+    function deregisterOperatorFromOperatorSets(address operator, uint32[] calldata operatorSetIds) external{}
```

Comment:
these can be directly called by the operator right? Or is it necessary to route these through service manager?

Reply:
These have to be routed through the service manager currently, but that will likely change post-UAM  

They won't be called directly by the operator on the service manager, though.  The flow would be

Registry -> ServiceManager -> AllocationManager

Reply:
what is UAM?

Reply:
universal access managemnent